Jenkinsfile

```diff
@@ -141,69 +178,76 @@ def uploadCli(architectures) {
     for (int i = 0; i < architectures.size(); i++) {
         def currentBuild = architectures[i]
         stage("Build and upload ${currentBuild.pkg}") {
-            buildAndUpload(currentBuild.goos, currentBuild.goarch, currentBuild.pkg, currentBuild.fileExtention)
+            buildAndUpload(currentBuild.goos, currentBuild.goarch, currentBuild.pkg, currentBuild.fileExtension)
         }
     }
 }
 
 def buildPublishDockerImages(version, jfrogCliRepoDir) {
+    def repo21Prefix = "${REPO_NAME_21}/ecosys-docker-local"
     def images = [
             // Pushing the second slim name for backward compatibility.
-            [dockerFile:'build/docker/slim/Dockerfile', names:['releases-docker.jfrog.io/jfrog/jfrog-cli', 'releases-docker.jfrog.io/jfrog/jfrog-cli-go']],
-            [dockerFile:'build/docker/full/Dockerfile', names:['releases-docker.jfrog.io/jfrog/jfrog-cli-full']]
+            [dockerFile:'build/docker/slim/Dockerfile', names:["jfrog/jfrog-cli", "jfrog/jfrog-cli-go"]],
+            [dockerFile:'build/docker/full/Dockerfile', names:["jfrog/jfrog-cli-full"]]
     ]
     for (int i = 0; i < images.size(); i++) {
         def currentImage = images[i]
         def primaryName = currentImage.names[0]
 
-        buildDockerImage(primaryName, version, currentImage.dockerFile, jfrogCliRepoDir)
-        pushDockerImageVersionAndRelease(primaryName, version)
+        def imageRepo21Name = "$repo21Prefix/$primaryName"
+        buildDockerImage(imageRepo21Name, version, currentImage.dockerFile, jfrogCliRepoDir)
+        pushDockerImageVersion(imageRepo21Name, version)
 
         // Push alternative tags if needed.
         for (int n = 1; n < currentImage.names.size(); n++) {
             def newName = currentImage.names[n]
-            // Create new tag.
-            sh """#!/bin/bash
-                docker tag $primaryName:$version $newName:$version
-            """
-            pushDockerImageVersionAndRelease(newName, version)
+            def currentRepo21Name = "$repo21Prefix/$newName"
+            buildDockerImage(currentRepo21Name, version, currentImage.dockerFile, jfrogCliRepoDir)
+            pushDockerImageVersion(currentRepo21Name, version)
         }
     }
-}
 
-def buildDockerImage(name, version, dockerFile, jfrogCliRepoDir) {
-    dir("$jfrogCliRepoDir") {
-        sh """#!/bin/bash
-            docker build --tag=$name:$version -f $dockerFile .
-        """
+    stage("Distribute cli-docker-images to releases") {
+        distributeToReleases("cli-docker-images2", version, "docker-images-rbc-spec.json")
```

Comment:
The '2' seems redundant because the release bundle has a suffix of the identifier